napari/layers/points/points.py

```diff
@@ -1328,6 +1348,7 @@ def _get_value(self, position) -> Union[None, int]:
         if len(view_data) > 0:
             displayed_position = [position[i] for i in self._dims_displayed]
             # Get the point sizes
+            # TODO: calculate distance in canvas space to account for canvas_size_limits
```

Comment:
Can you maybe add a comment here or elsewhere that without this implemented selection of points won't always work - just so we don't forget. I could see someone trying that and getting confused/ not remembering discussion on PR

Reply:
added some details!